README.md

```diff
@@ -481,6 +481,30 @@ end
 }
 ```
 
+#### Array type
+
+If you want to pass an array of primitive types, you can do this with the following:
```

Comment:
To nitpick, you're not passing anything. Aren't we just saying that array types are supported? Maybe just mention it above?